Source/Scene/Camera.js

```diff
@@ -1010,12 +1012,23 @@ define([
 
             var ray = this.getPickRay(mousePosition, pickGlobeScratchRay);
             rayIntersection = globe.pick(ray, scene, scratchRayIntersection);
-            if (defined(rayIntersection)) {
+
+            if (scene.pickPositionSupported) {
+                depthIntersection = scene.pickPositionWorldCoordinates(mousePosition, scratchDepthIntersection);
```

Comment:
Just wondering, are there other areas that commonly call `pickPosition`? Caching based on the window coordinates per-frame might not be a bad idea.